test/transcoding/SpecConstantComposite.ll

```diff
@@ -0,0 +1,135 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
```

Comment:
What do you think about adding `spirv-val` invocation here?

Reply:
Makes sense. I will add it.

Reply:
Added in 4a0fc3a.